File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/music_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class MusicVO:
    """音乐响应"""
    created_at: str = None
    updated_at: str = None
    music_id: str = None
    title: str = None
    description: str = None
    content_url: str = None
    format: str = None
    duration: int = None
    artist: str = None
    genre: str = None
    thumbnail_url: str = None
    status: str = None
    is_public: bool = None
    is_favorite: bool = None
    play_count: int = None
    like_count: int = None

File: sdkwork-app-sdk-typescript/src/types/music-vo.ts
/** 音乐响应 */
export interface MusicVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 音乐ID */
  musicId?: string;
  /** 标题 */
  title?: string;
  /** 描述 */
  description?: string;
  /** 音乐URL */
  contentUrl?: string;
  /** 格式 */
  format?: string;
  /** 时长(秒) */
  duration?: number;
  /** 艺术家 */
  artist?: string;
  /** 流派 */
  genre?: string;
  /** 缩略图URL */
  thumbnailUrl?: string;
  /** 状态 */
  status?: string;
  /** 是否公开 */
  isPublic?: boolean;
  /** 是否收藏 */
  isFavorite?: boolean;
  /** 播放次数 */
  playCount?: number;
  /** 点赞数 */
  likeCount?: number;
}
